### PR TITLE
Mvp triton post pr fixes

### PR DESCRIPTION
**Pull request by samouzegar10**
changes to match freezeman pr comment changes

### CODE REVIEW

File: triton-server/src/notification.ts [hunk 1]
```diff
@@ -67,9 +69,7 @@ export const sendNotificationEmail = async (
                         -   Readset blocked status count: ${
                             dataset.blocked_status_count
                         }<br/>
-                        -   Dataset latest released update date: ${formatDateAndTime(
-                            dataset.latest_release_update ?? new Date(),
-                        )}<br/>
+                        -   Dataset latest released update date: ${formatDateAndTime()}<br/>
```
Comment (nafiz1001) [inline triton-server/src/notification.ts:72]: I think it's better if you use `dataset.latest_release_update` first and then fallback to `formatDateAndTime()` if it's undefined for some reason.

File: triton-server/src/notification.ts [hunk 2]
```diff
@@ -141,15 +139,13 @@ const mockDataset: TritonDataset = {
...
+    const hours = new Date().getHours() - 1
+    cleanedDate.setHours(hours)
     return (
         cleanedDate.toLocaleDateString() +
-        " " +
-        cleanedDate.toLocaleTimeString()
+        "T" +
+        cleanedDate.toLocaleTimeString().split(" ")[0]
     )
 }
```
Comment (nafiz1001) [inline triton-server/src/notification.ts:151]: The function sounds general but it's doing oddly specific like settings hours back by 1. Maybe we should have an argument as Data object?

Comment (nafiz1001) [reply triton-server/src/notification.ts:151]: Wait a minute. It did have `Date` argument. Eh anyway.

File: triton-server/src/notification.ts (hunk 1 above)
Comment (nafiz1001) [reply triton-server/src/notification.ts:72]: `formatDateAndTime` could help with formatting if it accepts `Date`

File: triton-server/src/notification.ts (hunk 2 above)
Comment (samouzegar10) [reply triton-server/src/notification.ts:151]: this format is accepted by django, im sending a date in string format and django accepts it and returns the correct values, i've tested it locally so now its just missing to test on qc

**Review by nafiz1001 — APPROVED** [review]

**Pull request merged**